src/components/SortLink/SortLink.tsx

```diff
@@ -0,0 +1,39 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { SearchLink } from '../SearchLink/SearchLink';
+
+type TProps = {
+  field: string;
+};
+
+export const SortLink: React.FC<TProps> = ({ field }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  return (
+    <SearchLink
+      params={{
+        sort: sort !== field || (sort === field && !order)
```

Comment:
You could create a variable for this condition and use it here. This way would be easier to reed code.